server/src/test/resources/status.resource.test.runtime.properties

```diff
@@ -0,0 +1,11 @@
+druid.hidden.properties=druid.s3.accessKey,druid.s3.secretKey,druid.metadata.storage.connector.password
```

Comment:
this can be an array of string. It is better to rely on Jackson rather doing it via separators